### PR TITLE
Adds OSS::Attribute::TagArray component along with tests and documentation

### PR DESCRIPTION
**Pull request by Miexil**
Adds OSS::Attribute::TagArray component along with tests and documentation
Screenshot 2023-11-10 at 11 48 53
Screenshot 2023-11-10 at 15 57 58
Screenshot 2023-11-10 at 16 37 03

### Good PR checklist
- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [x] Added/updated tests
- [x] Added/updated documentation
- [ ] Migrated touched components to Glimmer Components
- [x] Properly labeled

### CODE REVIEW

File: addon/components/o-s-s/attribute/tag-array.stories.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        component:
+          'Member of the Attribute displays. The OSS::Attribute::TagArray displays all the tags that are provided in the @tags parameter.'
+      }
+    }
+  }
+};
+
+const defaultArgs = {
+  label: 'Fruits',
+  tags: ['watermelon', 'vodkamelon', 'whiskeymelon', 'tequilamelon']
```
Comment (JulienVannier66) [inline addon/components/o-s-s/attribute/tag-array.stories.js:41]: 😂

File: tests/integration/components/o-s-s/attribute/tag-array-test.ts
Comment (JulienVannier66) [inline tests/integration/components/o-s-s/attribute/tag-array-test.ts:1]: Can you add a test for the error if no label is setup?

**Review by JulienVannier66 — APPROVED** [review]
Otherwise LGTM

File: addon/components/o-s-s/attribute/tag-array.stories.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import hbs from 'htmlbars-inline-precompile';
+
+export default {
+  title: 'Components/OSS::Attribute::TagArray',
+  component: 'tag-array',
+  argTypes: {
+    label: {
```
[the remark this answers is not in the document]
Comment (Miexil) [reply addon/components/o-s-s/attribute/tag-array.stories.js:7]: 🤔 
it is defined at line 16

**Review by phndiaye — APPROVED** [review]

File: addon/components/o-s-s/attribute/removable-text.stories.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import hbs from 'htmlbars-inline-precompile';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Components/OSS::Attribute::RemovableText',
+  component: 'removable-text',
+  argTypes: {
```
Comment (JulienVannier66) [inline addon/components/o-s-s/attribute/removable-text.stories.js:7]: You forget to add `value` in the doc

**Review by JulienVannier66 — APPROVED** [review]
Otherwise LGTM

**Review by phndiaye — APPROVED** [review]

**Pull request merged**